src/in-house-liquidity/client.ts

```diff
@@ -24,7 +25,9 @@ export const getClient = async (): Promise<SiweApiClient> => {
         sessionDurationMs: SESSION_DURATION_MS,
         loginUrl: `${networkConfig.inHouseLiquidityURL}/auth/login`,
         clockUrl: `${networkConfig.inHouseLiquidityURL}/clock`,
-        timeout: networkTimeoutSecondsSelector(store.getState()) * 1000,
```

Comment:
I like that previously, the selector had "seconds" in the name. It makes me uneasy that we now use the word "timeout" to mean millis in some places and seconds in others

Reply:
I can update the dynamic config to include seconds

Reply:
I think you may have forgotten to rename the dynamic config

Reply:
I went ahead and did it